props/saber_fett263_buttons.h

```diff
@@ -2024,24 +2033,88 @@ SaberFett263Buttons() : PropBase() {}
   }
 #endif
 
-  void DoColorZoom() {
-    if (color_mode_ == COLOR_LIST) {
+  bool CancelShowColor() {
+    switch (menu_type_) {
```

Comment:
I seem to remember you having a similar list of cases elsewhere in the code.
Would it be possible to break it out into a helper function. Something like "bool isInMenu()" maybe?

Reply:
I think you're thinking of the MenuChoice(), MenuIncrement() and MenuUndo() functions although those are specific to actions taken in the menu not sure if a helper function would make it easier or harder to use.